packages/shared/nodes/UsfmParagraphNode.ts

```diff
@@ -10,16 +17,16 @@ export class UsfmParagraphNode extends UsfmElementNode {
   }
 
   static clone(node: UsfmParagraphNode): UsfmParagraphNode {
-    return new UsfmParagraphNode(node.__attributes, node.__data, node.__tag, node.__key);
+    return new UsfmParagraphNode(node.__attributes, node.__tag, node.__key);
   }
 
-  constructor(attributes: Attributes, data: unknown, tag?: string, key?: NodeKey) {
-    super(attributes, data, tag, key);
+  constructor(attributes: Attributes, tag?: string, key?: NodeKey) {
```

Comment:
nit: this construstor can be removed since it's the same as its parent.

Reply:
will do, thanks!